vhosting/vhosting_check.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+import argparse
+import urllib.request as req
+import os, re, requests, sys
+
+banners_dir = "./banners/"
+logs_dir = "./logs/"
```

Comment:
there's no point having a `logs` directory in the repo, it won't be writable by anyone if they use the version deployed at `/opt/share/utils`. i'd delete that `.gitkeep` and just have the script make it in a temporary directory or in the cwd